src/basic_commands/print_modes.md

```diff
@@ -439,3 +439,15 @@ e asm.syntax=att
 
 You can also check `asm.pseudo`, which is an experimental pseudocode view,
 and `asm.esil` which outputs [ESIL](../disassembling/esil.md) ('Evaluable Strings Intermediate Language'). ESIL's goal is to have a human-readable representation of every opcode semantics. Such representations can be evaluated (interpreted) to emulate effects of individual instructions.
+
+### Print gadgets
+
+Using print gadgets, Rizin allows you to display the output of a command anywhere on the screen. You can add a gadget using `pg`.
+
```

Comment:
@XVilka How about this?

```suggestion
### Print gadgets

In Rizin, visual gadgets allows the users to cast or display the output of a particular Rizin command anywhere on the screen while in Visual mode.

Using the commands under `pg` (print gadgets), we can add, remove and change the location of these visual gadgets. You can add a gadget using `pg`:
```

Reply:
Yes, but please also mention that it's ROP gadget.